open_academy/models/partner.py

```diff
@@ -0,0 +1,8 @@
+from odoo import models, fields
+
+
+class Partner(models.Model):
+    _inherit = "res.partner"
+
+    instructor = fields.Boolean(default=False)
+    session_ids = fields.Many2many("open.academy.session", readonly=True)
```

Comment:
As a good practice we always place for the name of the relation, the name of both models and the suffix `_rel`. The first name is current model and the second is related model.

```suggestion
    sessions_ids = fields.Many2many("open.academy.session", "res_partner_session_rel")
```

Please apply this comment in `open.academy.session` model to the `many2many` field.